pkg/config/toml/docs.toml

```
@@ -205,6 +205,8 @@ LimitMultiplier = '1.0' # Default
 LimitTransfer = 21_000 # Default
 # EstimateLimit enables estimating gas limits for transactions. This feature respects the gas limit provided during transaction creation as an upper bound.
 EstimateLimit = false # Default
+# SenderAddress is optional and can be set to a specific sender address for dynamic gas estimation. If it is not set, the actual "from" address for the transaction is used if available.
+SenderAddress = '' # Default
```

What does a default value of empty string mean? I think we want an example here:
```suggestion
SenderAddress = '0xabcd1234' # Example
```

---

This is an empty string by default because it does not have to be set. If it is not set (i.e. set to the empty string), then it is ignored, and the "from" address for the transaction is used for dynamic gas estimation instead. Also, setting this has no effect when `EstimateLimit` is set to false (which it is by default), so this should not be set by default

---

Please add an example value.

---

Understood; I have added an example value.